INSTALL.asciidoc

```diff
@@ -2,19 +2,30 @@
 pragmatic development environment on openSUSE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1.  install openQA package to get all the runtime dependencies
+1.  https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc#installation[install the openQA package] to get all the runtime dependencies
 +
 ---------------------
 sudo zypper in openQA
 ---------------------
++
+Alternatively installing the following packages will satisfy all dependencies at time of writing
++
+-----------------------------------------------------------------------
+zypper qemu tesseract optipng opencv tesseract-ocr sqlite3 mumble     \
+       libtheora-devel opencv-devel fftw3-devel libsndfile-devel      \
+       pkg-config libtool autoconf automake gcc-c++ make perl-JSON    \
+       perl-Mojolicious perl-Mojolicious-Plugin-AssetPack             \
+       perl-IPC-System-Simple perl-Exception-Class perl-Test-ANSIColor\
+       perl-Test-Compile perl-Test-More perl-Test-Simple perl-XML     \
+       perl-carp perl-Class-Accessor perl-Net-SSH2 perl-Crypt-DES
+-----------------------------------------------------------------------
```

Comment:
What is the correct way of doing this? Running `autogen.sh` I guess, which I may not have done. Instead I probably just ran `autoreconf` and then forgot about it. I have used SCons most recently for a build system and it works OK, but I don't fancy being the one to try porting OpenQA to it. There is also the predecessor to SCons: Cons, which is written in Perl.